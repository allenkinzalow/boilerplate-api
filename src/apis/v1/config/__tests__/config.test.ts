import app from "@app";
import request from "supertest";

describe("api/v1/config", () => {
  test("getConfig should return config", async () => {
    const res = await request(app).get("/api/v1/config");
    expect(res.body).toMatchSnapshot();
  });
});
