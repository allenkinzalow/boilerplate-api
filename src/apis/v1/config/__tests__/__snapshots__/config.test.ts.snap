// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`api/v1/config getConfig should return config 1`] = `
Object {
  "administrators_enabled": true,
}
`;
